Formula/abseil.rb

```diff
@@ -13,6 +14,11 @@ class Abseil < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "pkg-config" => :test
+
+  # Generate pkg-config files
+  # https://github.com/abseil/abseil-cpp/commit/eb317a701b83bf9a4f2a035d75747a3d76a48324
+  patch :DATA
```

Comment:
Can't we just use the upstream commit to generate the patch? I know it changes more than we need, but large embedded patches can get a bit fragile.

Also, the comment should probably say when the patch can be removed.

Reply:
> Can't we just use the upstream commit to generate the patch?

No, there are too many conflicts

> Also, the comment should probably say when the patch can be removed.

I'm not sure which release this will be included in since they cherry-pick commits for releases

> large embedded patches can get a bit fragile.

Admittedly, this isn't urgent, so I'll close this PR and we can revisit this if necessary

Reply:
Sure; the change seems like a good idea though. Hopefully upstream include this in a stable release sooner rather than later.